docs/reference/errors-and-warnings/NU1202.md

```diff
@@ -18,4 +18,5 @@ f1_keywords:
 A dependency package doesn't contain any assets compatible with the project.
 
 ### Solution
-Change the project's target framework to one that the package supports.
\ No newline at end of file
+- Change the project's target framework to one that the package supports.
+- Give our new experimental one-click tool, [NuGetSolver](https://devblogs.microsoft.com/nuget/introducing-nugetsolver-a-powerful-tool-for-resolving-nuget-dependency-conflicts-in-visual-studio/), a try. This tool can automatically fix this problem if you’re using VS2022 with nuget.org. Please provide us with feedback. Based on the feedback received, we will decide whether to officially support this new tool and incorporate it into the NuGet package manager.
```

Comment:
I thought NuGetSolver did not have any compatibility logic?

Reply:
NU1202 is covered by NuGetSolver, teams discussion is [here](https://teams.microsoft.com/l/message/19:f872be6c5c074cc29bfcd028926261c8@thread.skype/1683076545967?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ad34ac55-6785-4e7c-82c3-7ba405d30c3c&parentMessageId=1683065832251&teamName=NuGet&channelName=NuGetSolver%20development%20(NuFix)&createdTime=1683076545967).
As posted in blog post it works with multi-tfm projects, it can solve conflict for at least one tfm, maybe not all of them.